src/resources/i18n/translations/en.json

```diff
@@ -311,31 +310,31 @@
 	"Export OPML": "Export OPML",
 	"External Storage Enabled": "External storage card on",
 	"FAQ": "FAQ",
-	"FOSS purchase alert text": "As per F-Droid FOSS guidelines, visit the https://podverse.fm to renew your Podverse premium.",
-	"Feature Request": "Feature request",
+	"FOSS purchase alert text": "As per F-Droid FOSS guidelines, visit the https://podverse.fm to renew your Podverse Premium.",
+	"Feature Request": "Feature Request",
 	"Features": "Features",
 	"Filter": "Filter",
-	"Free Trial Expired": "Gratis Podverse premium period expired",
-	"Free": "Gratis",
+	"Free Trial Expired": "Podverse Premium period expired",
+	"Free": "Free",
```

Comment:
```suggestion
	"Free": "Gratis",
```
or "At no cost"

Reply:
`"Free Trial Expired": "Podverse Premium period expired",`

Can you change it to:

`"Free Trial Expired": "Podverse Premium trial period expired",`? We want to distinguish between trial periods ending, and paid memberships ending.

"Free": "At no cost",`

For English, we want to go with "Free" here. In longer more descriptive text sections we're open to going with gratis in English.

Reply:
Never mind, you've convinced me not to go with the negative "trial" association. Can you change it to "Podverse Premium expired"? This text is used as a header in an alert, followed by a description text, so I don't think the header has to be as descriptive. 

We'd still like "Free" in this case for English. It only gets rendered in this one place on the "Free vs Premium" account comparison table.

![image](https://user-images.githubusercontent.com/2262919/153319944-06495a98-ae6d-47a8-a250-87a7925df6a4.png)